answer.txt

```diff
@@ -0,0 +1,11 @@
+• What is a gitignore file and what should it include for a Java project?
+	So the gitignore file contains the ignore rules. Place the gitignore file in the main directory of the Eclipse Project.
+
+• What does fast-forward in git mean?
+	It means that when there is a direct linear path from the source branch to the target branch.
```

Comment:
> It
Answer should be formal
> that when
Incorrect wording.

Also fast-forwarding is an action that you can perform in git, which is literally move the head from its current location to the end of the branch we are merging. The term `merging` is missing from your answer.